From: Simon McVittie <smcv@collabora.com>
Date: Mon, 27 Feb 2023 12:40:05 +0000
Subject: tests: Ensure non-root users have access to libcap tools

On Debian systems, by default only root has /{usr/,}sbin in PATH.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: https://github.com/ostreedev/ostree/pull/2821
---
 tests/libtest.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/libtest.sh b/tests/libtest.sh
index 264094b..963bc92 100755
--- a/tests/libtest.sh
+++ b/tests/libtest.sh
@@ -32,6 +32,9 @@ else
 fi
 . ${test_srcdir}/libtest-core.sh
 
+# Make sure /sbin/capsh etc. are in our PATH even if non-root
+PATH="$PATH:/usr/sbin:/sbin"
+
 # Array of expressions to execute when exiting. Each expression should
 # be a single string (quoting if necessary) that will be eval'd. To add
 # a command to run on exit, append to the libtest_exit_cmds array like
